deeppavlov/models/ranking/bilstm_gru_siamese_network.py

```diff
@@ -48,7 +48,7 @@ class BiLSTMGRUSiameseNetwork(BiLSTMSiameseNetwork):
             Only required if ``triplet_loss`` is set to ``True``.
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, **kwargs) -> None:
         super(BiLSTMGRUSiameseNetwork, self).__init__(*args, **kwargs)
```

Comment:
If a method does the same thing as its ancestor, it can be skipped

Reply:
corrected